File: Python programming/LAB 9/8.py
# Write a recursive function to obtain length of a given string.

str=str(input("Enter a string:"))

def length(str):
    if(str==""):
        return 0    #---------
    else:
        return(1+length(str[1:]))

print(f"The input string is:{str}")
print(f"The length of string is:{length(str)}")

File: c programming/Unit 1/8.c
/*write a program to convert fahrenheit to degree and vice versa.*/

#include<stdio.h>
void main(){
    float temp,degree,fahrenheit;
    printf("Enter the temperature:");
    scanf("%f",&temp);
    fahrenheit=(1.8*temp)+32;
    degree=(0.5556)*(temp-32);
    printf("Temperature in fahrenheit is:%.2f\n",fahrenheit);
    printf("Temperature in degree is:%.2f",degree);
}